ports/espressif/bindings/espulp/__init__.c

```diff
@@ -80,6 +82,7 @@ STATIC const mp_rom_map_elem_t espulp_module_globals_table[] = {
     // module classes
     { MP_ROM_QSTR(MP_QSTR_ULP), MP_OBJ_FROM_PTR(&espulp_ulp_type) },
     { MP_ROM_QSTR(MP_QSTR_ULPAlarm), MP_OBJ_FROM_PTR(&espulp_ulpalarm_type) },
+    { MP_ROM_QSTR(MP_QSTR_ULPArch),    MP_ROM_PTR(&espulp_ulparch_type) },
```

Comment:
I'd write this out. The ULP prefix isn't really needed because it is in the module name. The alarm has it as an artifact of it originating in alarm and to match the other alarm names.

```suggestion
    { MP_ROM_QSTR(MP_QSTR_Architecture),    MP_ROM_PTR(&espulp_architecture_type) },
```